### ISSUE TITLE
[1.12.2] Too many guards

### ISSUE THREAD

**Issue by NotAVerb** [ToroCraft/ToroQuest#187]
Having two to three times as many guards as there are villagers in a village is silly.
I'm referring to the dynamic guards of course.

**Comment by MLMII** [comment]
A option to control their spawnrates would be nice.

**Comment by Rutes** [comment]
I agree the guards and sentries are too numerous - this happens usually when they can't kill a zombie because it's underground or in a tree or something...
Similar issue reported long ago:
https://github.com/ToroCraft/ToroQuest/issues/169

**Comment by Gamerbolts** [comment]
I am encountering this issue as well, because of Advent of Ascension. The Guards can't hit those mobs and continue to spawn like crazy.

**Comment by kreezxil** [comment]
@frodare has this been fixed yet? @Wing01

**Comment by frodare** [comment]
@kreezxil  I haven't worked on this project in a long time and due to the size of it probably won't do much more with it in the future (at least in its current form).  If you check the comments on curse however, someone else said they were fixing a lot of these issues and adding more features, so hopefully that pans out.
